style.css

```
@@ -1,93 +1,757 @@
+  height: 39px;
+  margin-left: 16px;
+  line-height: 44px;
+}
+
+.tonic {
+  font-weight: bold;
+  font-size: 32px;
+  font-family: 'Poppins', sans-serif;
+  color: #172b4d;
+  font-style: normal;
+}
+
+.title-items {
+  margin-left: 16px;
+  margin-top: 20px;
+  display: flex;
+  padding: 0;
+  gap: 20px;
+  width: 250px;
+}
+
+.canopy {
+  list-style: none;
+  width: 70px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.back-end {
+  width: 90px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.date {
+  width: 29px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.daily {
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 16px;
+}
+
+.tech-skills {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 203px;
+  list-style: none;
+  margin-left: 16px;
+}
+
+.my-html {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 53px;
+  height: 24px;
+}
+
+.my-css {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 45px;
+  height: 24px;
+}
+
+.js {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 89px;
+  height: 24px;
+}
+
+.project {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 16px;
+  height: 72px;
+  width: 295px;
+  left: 0;
+  top: 216px;
   border-radius: 0;
-  margin-top: 24px;
+}
+
+.see {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 15px;
+  gap: 12px;
+  width: 140px;
+  height: 55px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background: #fff;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+}
```

- [ ] As requested by the previous reviewer kindly make sure to style correctly the `see project` button in order to look like the provided Figma design. :wink: 
 
   ### PLease check out my suggestions:

```suggestion
.see {
  background: #fff;
  border: 1px solid #396df2;
  border-radius: 7px;
  padding: 7px 5px;
  color: #396df2;
  font-family: 'Poppins', sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 15px;
  line-height: 24px;
}
```